service_test.go

```diff
@@ -143,427 +144,433 @@ var mockTransactions = []Nip47Transaction{
 }
 var mockTransaction = &mockTransactions[0]
 
-// TODO: split up into individual tests
-func TestHandleEvent(t *testing.T) {
+func TestBackwardsCompatibility(t *testing.T) {
+	// Test without adding a single permission
```

Comment:
can this be done in this PR?

Reply:
Actually, this is going to be cleaned up in the self-custodial version. I added a TODO in the Wails PR. Nice!

Reply:
Should I just remove this completely then?

Reply:
Sounds good!